src/index.less

```diff
@@ -89,6 +90,21 @@ header .navbar.navbar-static-top {
   }
 }
 
+.sort-form,
+.sort-form > .sort-group {
+  margin-bottom: 0;
+}
+.pager {
+  & > li > a {
+    cursor: pointer;
+    margin: 0 2px;
+  }
+}
```

Comment:
Sorry I could not resist my OCD alarm. This is just to clean up the layout just a tiny bit.

**before**
![image](https://user-images.githubusercontent.com/2130305/99887590-c8e8d600-2c45-11eb-8966-2bc12231aa16.png)


**now:**
![image](https://user-images.githubusercontent.com/2130305/99887570-a8208080-2c45-11eb-8d76-d26d83970bf5.png)